business/istio_config.go

```diff
@@ -79,12 +79,6 @@ var apiToVersion = map[string]string{
 	kubernetes.RbacGroupVersion.Group:       kubernetes.ApiRbacVersion,
 }
 
-const (
-	MeshmTLSEnabled          = "MESH_MTLS_ENABLED"
```

Comment:
Is this removed / changed ?
Because I understand this value is sent to the UI. Right ?
I ask because the Istio Wizard use this value to pre-set the TLS value.